### ISSUE TITLE
Is the valuation completely ignoring storage and solar or is it accounted for but not modelled somehow?

### ISSUE THREAD

**Issue by SpaceMonkeyForever** [ARKInvest/ARK-Invest-Tesla-Valuation-Model#27]
I see this note in their article:

> Note: We do not model Tesla’s utility energy storage or solar business in our models. We also have not modeled bitcoin assumptions in our model. For ARK’s work on bitcoin as corporate cash, please download our latest Big Ideas presentation.

Does that mean their valuations here are not even taking into account energy storage and solar? or are you accounting for it somehow without fully modelling it e.g. by using someone else's forecasts for storage and solar?

**Comment by joekendal** [comment]
Nothing of what Ark Invest has put out this week makes any sense. Tesla is a bubble of epic proportions. I used to have a modicum of respect for Cathie Wood but now it's almost embarrassing to watch this.

**Comment by Blondbadass** [comment]
Hello pls can u help me with invest bot ?

**Comment by vnktsh** [comment]
@joekendal Thanks so much for such a valuable, critical and informative rebuke. We're enlightened.

**Comment by knuppe** [comment]
> Nothing of what Ark Invest has put out this week makes any sense. Tesla is a bubble of epic proportions. I used to have a modicum of respect for Cathie Wood but now it's almost embarrassing to watch this.

To criticize, you must have some better model, share your knowledge with us.

**Comment by joekendal** [comment]
@knuppe yes, I can only imagine Warren Buffet thinking _I should run a monte carlo simulation then that will give me the stock's value_. This guy is clueless obviously 🤦‍♂️  go and write some macros in Excel 😂  This guy man 😭

**Comment by rayneman** [comment]
Canaccord Genuity analyst predicted generation and storage unit could yield $8 billion of revenue by 2025 and creating an Apple-esque ecosystem of energy products.
